.github/workflows/project-meta-sync.yml

```
@@ -21,26 +21,27 @@ jobs:
-          id: app-token
-          uses: actions/create-github-app-token@v2
-          with:
-            app-id: ${{ vars.LS_APP_ID }}
-            private-key: ${{ secrets.LS_APP_PRIVATE_KEY }}
-            # Optionally scope permissions tighter (inherits installation perms by default)
-            # permission-issues: read
-            # permission-pull-requests: read
-            # permission-projects: write
+      - name: Create GitHub App installation token
+        id: app-token
+        if: vars.LS_APP_ID && secrets.LS_APP_PRIVATE_KEY
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)